src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -711,6 +711,74 @@ def test_sql_db_operation_mgmt(self, resource_group, resource_group_location, se
                  .format(resource_group, server, database_name, ops[0]['name']))
 
 
+class SqlManagedInstanceOperationMgmtScenarioTest(ScenarioTest):
+
+    @record_only()
+    def test_sql_mi_operation_mgmt(self):
+        managed_instance_name = self.create_random_name(managed_instance_name_prefix, managed_instance_name_max_length)
+        admin_login = 'admin123'
+        admin_password = 'SecretPassword123'
+
+        is_playback = os.path.exists(self.recording_file)
+        if is_playback:
+            subnet = '/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/AndyPG/providers/Microsoft.Network/virtualNetworks/prepare-cl-nimilj/subnets/default'
+        else:
+            subnet = '/subscriptions/a8c9a924-06c0-4bde-9788-e7b1370969e1/resourceGroups/AndyPG/providers/Microsoft.Network/virtualNetworks/prepare-cl-nimilj/subnets/default'
```

Comment:
Please do not hardcode resources, it causes massive problems for other engineers when they need to re-record your test. Follow example of other MI CLI tests that create subnet at runtime.

Reply:
Sure, will change.